authenticator/pyproject.toml

```diff
@@ -20,7 +20,7 @@ classifiers = [
 ]
 requires-python = ">=3.10"
 dependencies = [
-    "jupyterhub<5",
+    "jupyterhub==5",
```

Comment:
I don't think you want the plugin to only allow version 5.0.0 exactly, which is what that does. Also, the plugin should continue to work fine with earlier versions of jupyterhub, so I think leaving this at `jupyterhub<6` is fine until we know for sure that we did something that broke compatibility.

Reply:
Huh.  I thought that meant "Anything with major version 5"